syntax = "proto3";
package protobufMsg;
message HeartBeat {
  int32 actionId = 1;
}
